sklearn/datasets/tests/test_base.py

```diff
@@ -186,12 +186,12 @@ def test_loader(loader_func, data_shape, target_shape, n_target, has_descr,
 
 
 @pytest.mark.parametrize("loader_func, data_dtype, target_dtype", [
-    (load_breast_cancer, np.float64, np.int64),
+    (load_breast_cancer, np.float64, np.int),
```

Comment:
`np.int` is [being deprecated in numpy](https://github.com/numpy/numpy/pull/6103). Does the following work?

```suggestion
    (load_breast_cancer, np.float64, np.int_),
```

Reply:
Actually, my use of `np.int` was mistaken anyway. The data-loading function uses plain (builtin) `int`, not `np.int` as I originally claimed:

https://github.com/scikit-learn/scikit-learn/blob/8f09e332b663768f23a75448f7e71c47cd14a785/sklearn/datasets/_base.py#L267

I propose switching to that (PR updated), so that the data-loading function and the test are consistent. If you'd prefer `np.int_`, please let me know. Sorry about the confusion.


On a side note: `grep -R 'np\.int[^_1368a-z]' sklearn/*` gives me ~100 occurrences of `np.int` in the codebase which will probably need to be updated at some point, along with the other deprecated types.

Reply:
`int` is works as well.